SpatialGDK/Source/SpatialGDKFunctionalTests/Private/SpatialFunctionalTest.cpp

```diff
@@ -817,6 +875,67 @@ void ASpatialFunctionalTest::OnReplicated_bFinishedTest()
 	}
 }
 
+void ASpatialFunctionalTest::LogAndClearRequireHandler()
+{
+	TArray<FSpatialFunctionalTestRequire> RequiresOrdered = RequireHandler.GetAndClearStepRequires();
+
+	ASpatialFunctionalTestFlowController* FlowController = GetLocalFlowController();
+	if (FlowController)
```

Comment:
Why did you move this logging code out of RequireHandler?

Reply:
Because we need to propagate the require messages accross multiple processes, we needed to multicast the logging functionality. Unfortunately, RequireHandler isnt an Actor and the data structure which stores the require messages are private, so I had to move this logic into the ASpatialFunctionalTest to have the function multicasted.

Reply:
On top of this, it seemed to make sense in a way as the old LogAndClearRequire would need to access the flow controller, which it does via `OwningTest->GetFlowController()` (essentially going up a hierarchy and going back down again, so it felt somewhat intuitive to move this logic up a hierarchy into the test actor where both the RequireHandler and FlowController objects are available

Reply:
So RequireHandler can't call `ServerNotifyLogRequireMessages` ?

Reply:
Ahh let me take another look, that might solve your other comment as well

Reply:
You're right, we can keep this functinoality in one place by using `ServerNotifyLogRequireMessages` from RequireHandler. My bad on this I completely missed it, the fix is coming!